purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallColor.kt

```diff
@@ -54,6 +58,15 @@ data class PaywallColor(
             null
         },
     )
+
+    constructor(@ColorInt colorInt: Int) : this(
```

Comment:
Can we add tests for this?

Reply:
Added a test for this constructor. I plan on adding a bunch of tests after this gets merged